bindata/network/openshift-sdn/000-ns.yaml

```diff
@@ -9,3 +9,15 @@ metadata:
   annotations:
     openshift.io/node-selector: "" #override default node selector
     openshift.io/description: "OpenShift SDN components"
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  # NOTE: this namespace is the portal for host network traffic
+  name: openshift-host-network
+  labels:
+    openshift.io/run-level: "0"
+    openshift.io/cluster-monitoring: "true"
+  annotations:
+    openshift.io/node-selector: ""
+    openshift.io/description: "OpenShift SDN Host Network Traffic"
```

Comment:
nit: I would add a more detailed description here.

Reply:
what would you like for this to say? 
`Namespace created for enabling network policies for host network traffic` sounds good?